### DISCUSSION TITLE
HX8357D in parallel mode?

### DISCUSSION THREAD

**Discussion by neilenns** [Bodmer/TFT_eSPI#2358] [New processor/display support or capability request]
Has anyone ever tried to get the [HX8357D chipset controller](http://www.adafruit.com/datasheets/HX8357-D_DS_April2012.pdf) working in parallel instead of SPI mode? Based on the readme it appears it's the one of the three HX chipsets that doesn't currently support parallel mode on RP2040s, although in theory it could?

**Comment by Bodmer** [comment]
I do not have one f those displays but it should work fine in parallel mode with the RP2040.

**Reply by neilenns** [reply]
Ok thanks. I'll probably order one next month. If I get it working I'll let you know so the docs can update to indicate it's supported on the RP2040 in parallel mode as well.
